gruntfile.coffee

```diff
@@ -173,7 +177,7 @@ module.exports = ->
   @registerTask 'clean', ['remove']
   @registerTask 'lint', ['coffeelint']
   @registerTask 'build', ['clean', 'coffeelint', 'concat:dist', 'string-replace:strip_for_dist', 'coffee:dist','string-replace:strip_asserts', 'uglify:dist']
-  @registerTask 'test', ['coffeelint', 'mochaTest:all']
+  @registerTask 'test', ['coffeelint', 'mochaTest:all', 'mochaTest:examples']
```

Comment:
I realize that but I don't think this should be part of the suite of tests to confirm `finitedomain` works for reasons mentioned before.